.rusty-hook.toml

```diff
@@ -0,0 +1,6 @@
+[hooks]
+pre-commit = "cargo test"
+pre-push = "cargo test && cargo check && cargo clippy && cargo fmt -- --check"
```

Comment:
These run pre-push but they don't stop the push when warnings are generated by clippy for example.